File: zul/src/archive/web/js/zul/box/Vlayout.js
/* Vlayout.js

	Purpose:
		
	Description:
		
	History:
		Fri Aug  6 12:37:19 TST 2010, Created by jumperchen

Copyright (C) 2010 Potix Corporation. All Rights Reserved.

This program is distributed under GPL Version 3.0 in the hope that
it will be useful, but WITHOUT ANY WARRANTY.
*/
/**
 * A vertical layout.
 * <p>Default {@link #getZclass}: z-vlayout.
 * @since 5.0.4
 */
zul.box.Vlayout = zk.$extends(zul.box.Layout, {
	getZclass: function () {
		return this._zclass == null ? "z-vlayout" : this._zclass;
	},
	isVertical_: function () {
		return true;
	}
});

File: zul/src/main/java/org/zkoss/zul/Vlayout.java
/* Vlayout.java

{{IS_NOTE
	Purpose:
		
	Description:
		
	History:
		Aug 6, 2010 12:46:39 PM , Created by jumperchen
}}IS_NOTE

Copyright (C) 2010 Potix Corporation. All Rights Reserved.

{{IS_RIGHT
}}IS_RIGHT
*/
package org.zkoss.zul;

/**
 * A vertical layout
 *<p>Default {@link #getZclass}: z-vlayout.
 * @author jumperchen
 * @since 5.0.4
 */
public class Vlayout extends Layout {
	public String getZclass() {
		return _zclass == null ? "z-vlayout" : _zclass;
	}
}
